cdap-api/src/main/java/co/cask/cdap/api/dataset/table/Tables.java

```diff
@@ -89,7 +89,7 @@ public static void createTable(ApplicationConfigurer configurer, String datasetN
   /**
    * Creates properties for {@link Table} or {@link Table} data set instance.
    * @param level level on which to detect conflicts in changes made by different transactions
-   * @param ttl time to live for data written into a table, in ms. {@link #NO_TTL} means unlimited
+   * @param ttl time to live for data written into a table, in seconds. {@link #NO_TTL} means unlimited
```

Comment:
and actually I am surprised that this is an int. With 2B as the max int and ~86K seconds in a day, that is roughly 60 years. Probably enough. But when this was in milliseconds, that is less than a month.